images/router/haproxy/conf/haproxy-config.template

```diff
@@ -101,6 +97,12 @@ frontend public
   tcp-request inspect-delay 5s
   tcp-request content accept if HTTP
 
+  # Prevent vulnerability to POODLE attacks
+  {{ if matchPattern "true|TRUE" (env "ROUTER_DISABLE_SSLV3_PASSTHROUGH" "") }}
+  acl sslv3 req.ssl_ver 3
+  tcp-request content reject if sslv3
+  {{ end }}
+
```

Comment:
Doesn't make sense to do it for the `public` frontend ... we should do this for the secure traffic instead - so that should be in the `public_ssl` frontend. 
But that said ... because we first need to handle passthrough traffic  (we should let the pod control that for packets/traffic we pass through) we should probably do this check just before we use the `*sni` backends.